source/server/http/admin.cc

```diff
@@ -401,18 +401,34 @@ Http::Code AdminImpl::handlerStats(absl::string_view url, Http::HeaderMap& respo
     all_stats.emplace(gauge->name(), gauge->value());
   }
 
+  for (const Stats::ParentHistogramSharedPtr& histogram : server_.stats().histograms()) {
+    std::vector<std::string> summary;
+    const std::vector<double>& supported_quantiles_ref =
+        histogram->intervalStatistics().supportedQuantiles();
+    for (size_t i = 0; i < supported_quantiles_ref.size(); ++i) {
+      summary.push_back(fmt::format("P{}({},{})", 100 * supported_quantiles_ref[i],
```

Comment:
@mattklein123 capturing your question from the previous PR, so that we can continue the discussion.
Do we think cumulative has much benefit? I can't imagine it being very useful from a debugging perspective so I wonder if we should just drop it here? Thoughts?
Here is the current format
`cluster.time.upstream_rq_time: P0(0,0) P25(0,0) P50(0,0) P75(0,1.015) P90(1.06387,1.07194) P95(1.08525,1.09093) P99(2.04331,2.07759) P99.9(4.08333,23) P100(23,140)`